charts/cluster-autoscaler-chart/templates/deployment.yaml

```diff
@@ -156,10 +156,15 @@ spec:
                   name: {{ default (include "cluster-autoscaler.fullname" $) $value.name }}
                   key: {{ required "Must specify key!" $value.key }}
           {{- end }}
-          {{- if .Values.envFromSecret }}
+          {{- if or .Values.envFromSecret .Values.envFromConfigMap }}
           envFrom:
+          {{- if .Values.envFromSecret }}
             - secretRef:
                 name: {{ .Values.envFromSecret }}
+          {{- else if .Values.envFromConfigMap }}
```

Comment:
Do we want this to be an `else if` could a user not conceivably want to define both `envFromSecret` and `envFromConfigMap`?

Reply:
Allowing both is better, you're right. I'll change it.